doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -206,6 +175,42 @@ which provides the isolation of Operating System kernels.
 The architecture must support a way to isolate the compute resources of the
 infrastructure itself from the workloads compute resources.
 
+
```

Comment:
The updated ToC is:
- 3.2.1.1 Container Runtime Services
- 3.2.1.2 CPU Management
- 3.2.1.3 Memory and Huge Pages Resources Management
- 3.2.1.4 Hardware Topology Management
- 3.2.1.5 Node Feature Discovery
- 3.2.1.6 Device Plugin Framework
- 3.2.1.7 Hardware Acceleration
- 3.2.1.8 Scheduling Pods with Non-resilient Applications

The numbering below is inconsistent with the new ToC.

Reply:
> The numbering below is inconsistent with the new ToC.

Yes, numbering for sections in text is coming in #2147 , #2148 , #2150 , #2151 , #2152 , #2153 and #2154 .